extensions/amp-story/1.0/amp-story-tooltip.js

```diff
@@ -206,14 +208,18 @@ export class AmpStoryTooltip {
    * @private
    */
   attachTooltipToEl_(clickedEl) {
-    const {href} = parseUrlDeprecated(clickedEl.getAttribute('href'));
+    const elUrl = clickedEl.getAttribute('href');
+    user().assert(isProtocolValid(elUrl),'%s: The url is invalid', TAG);
+    const iconUrl = clickedEl.getAttribute('data-tooltip-icon');
+    user().assert(isProtocolValid(iconUrl), '%s: The url is invalid', TAG);
```

Comment:
If I recall correctly, this is going to get stripped from the production build, and the `isProtocolValid(iconUrl)` won't be performed.

How about
```
if (!isProtocolValid(iconUrl)) {
  user().error(....);
  return;
}
```

Reply:
Sorry for the double comment. Also, what happens if no `data-tooltip-icon` is provided? Is it triggering the error?

Reply:
It won't. An empty url is a valid protocol:

https://github.com/ampproject/amphtml/blob/ab5bd041b6aded0f1b84a2aafea3c0c7005eb151/src/url.js#L412-L415